caffe2/python/operator_test/mkl_alexnet_test.py

```diff
@@ -0,0 +1,337 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+import unittest
+
+import numpy as np
+from caffe2.proto import caffe2_pb2
+from caffe2.python import cnn, core, workspace, test_util
+
+
+@unittest.skipIf(not workspace.C.has_mkldnn, "Skipping as we do not have mkldnn.")
+class TestAlexNet(test_util.TestCase):
+    bs = 32
+    layerwisetiming = False
+
+    def testMKLAlexNet(self):
```

Comment:
nit: I think currently for python we use the following style:

function_name

test_blah_blah